apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -186,6 +192,13 @@ export function useApproveCallback(
     ],
   )
 
+  const approveNoCheck = useCallback(
+    async (overrideAmountApprove?: bigint) => {
+      return approve(overrideAmountApprove, false)
+    },
+    [approve],
+  )
```

Comment:
hmm it's getting worse adding these bypass functions

Reply:
if without state check, what if the user double clicked on the approve button? Does it trigger twice the approve tx?

Reply:
but it depends use case, on current universal router case, it triggered by hooks automaticly

Reply:
it's depends on use case, on universal router case, it only triggered by hooks, not triggered by user event